www/result.inc

```diff
@@ -177,26 +177,28 @@ $page_description = "Website performance test result$testLabel.";
                           $friendlyMedianMetric = htmlspecialchars($median_metric);
                       }
                       if ($testResults->countRuns() > 1) {
-                          echo '<em>(Based on Median Run by ' . $friendlyMedianMetric . ')</em>';
+                          echo '<em>(Based on Median Run by: 
+                          <details><summary>' . $friendlyMedianMetric . '</summary>';
+                          echo "<ul>";
+
+                          $url = strtok($_SERVER["REQUEST_URI"], '?');
+                        foreach ($medianMetrics as $key => $value) {
```

Comment:
I know it's unrelated to your change but the indentation looks a bit off, see if `composer format` fixes it